userspace/libsinsp/sinsp_suppress.cpp

```diff
@@ -55,6 +67,9 @@ int32_t libsinsp::sinsp_suppress::process_event(scap_evt *e)
 	// we need to check the comm, which might also update the set
 	// of suppressed tids.
 
+	uint64_t tid;
```

Comment:
As above, can you leave a a self review of the changes?

Reply:
The changes here are just to avoid any misaligned access to memory (thanks to ASAN) :)